### ISSUE TITLE
outline broken for radio button when using voiceover

### ISSUE THREAD

**Issue by DeepakSap14** [SAP/fundamental-styles#1640]
**Describe the bug**
Outline for voiceover is broken for radio button. see screenshot below.

> Is this issue related to a specific component?
Radio button

> What version of the Fundamental Library Styles are you using?
0.11
> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

> What front-end framework are you implementing in, e.g., Angular, React, etc.?
Angular

**To Reproduce**
1. Go to 'https://sap.github.io/fundamental-styles/?path=/docs/components-forms-radio--primary'
2. start voiceover
3. tab on radio

**Expected behavior**
outline should be proper as shown in below screenshot.

**Screenshots**
When VoiceOver is on, the outline is broken.
<img width="176" alt="Screenshot 2020-09-16 at 16 39 19" src="https://user-images.githubusercontent.com/8318592/93345199-3558ea80-f83b-11ea-981c-2c663148998f.png">
But should be like this
<img width="324" alt="Screenshot 2020-09-16 at 16 41 22" src="https://user-images.githubusercontent.com/8318592/93345448-7f41d080-f83b-11ea-9494-56be30bd2a36.png">

**Desktop (please complete the following information):**
 - Browser [e.g. chrome, safari] 'chrome'

**Smartphone (please complete the following information):**

**Additional context**
